libs/ui/src/system_call_api.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import { Optional } from '@votingworks/basics';
+import { useMutation } from '@tanstack/react-query';
+import { SystemCallApi } from '@votingworks/types';
+import * as grout from '@votingworks/grout';
+
+// Unable to use `grout.Client<SystemCallApi>` directly due to some mismatched
+// type inference with `grout.AnyMethods`, so copying the `grout.Client`
+// definition here for now:
+export type SystemCallApiClient = {
```

Comment:
Not 100% sure, but you might be able to use `type SystemCallApiClient = grout.Client<SystemCallApi>` here -- Jonah pointed out that the only reason it didn't work in the UiStrings case was that `UiStringsApi` wasn't built with `grout.buildApi()`.

Reply:
I've sketched a fix for this issue here: https://github.com/votingworks/vxsuite/pull/4215.

The important thing is to use the `grout.createApi` call to infer the type of the API rather than separately defining an interface in libs/types.